lib/PreviewProvider.js

```diff
@@ -36,6 +38,7 @@ const DEFAULT_OPTIONS = {
 function PreviewProvider(options = {}) {
   this.options = Object.assign({}, DEFAULT_OPTIONS, options);
   this._onPrefChange = this._onPrefChange.bind(this);
+  this._tippyTopProvider = new TippyTopProvider();
```

Comment:
Since `PreviewProvider` is a singleton, there will also be only one instance of `TippyTopProvider`. Since `PreviewProvider` is the only consumer, it makes sense to only instantiate it here IMO